src/test/util_tests.cpp

```diff
@@ -1696,4 +1697,105 @@ BOOST_AUTO_TEST_CASE(test_spanparsing)
     BOOST_CHECK_EQUAL(SpanToStr(results[3]), "");
 }
 
+namespace {
+
+struct Tracker
+{
+    //!< Points to the original object (possibly itself) we moved/copied from
+    const Tracker* origin;
+    //!< How many copies where involved between the original object and this one (moves are not counted)
```

Comment:
nit: These comments should not have ‘<‘, which is for in-line docs. http://www.doxygen.nl/manual/docblocks.html#memberdoc

Reply:
Thanks, fixed!